bin/laminas

```diff
@@ -23,8 +27,33 @@ if (file_exists($a = __DIR__ . '/../../../autoload.php')) {
     exit(1);
 }
 
+$projectRoot = dirname($a) . '/../';
+
 // Set the main application directory as the current working directory
-chdir(dirname($a) . '/../');
+chdir($projectRoot);
+
+$app = (new ApplicationFactory())();
+$definition = $app->getDefinition();
+$output = new ConsoleOutput();
+$containerNotFoundMessage = '';
+try {
+    $input = new ArgvInput();
+    $input->bind($definition);
+    $container = (new ContainerResolver($projectRoot))->resolve($input);
+    $app = (new ApplicationConfigurator())($app, $container);
+} catch (\Symfony\Component\Console\Exception\RuntimeException $exception) {
+    // Suppress as it will be part of the application output
```

Comment:
I'm confused here - wouldn't we want to report the issue with configuring the application and exit early for a configuration error like this? Or will symfony/console exit with an error status at this point anyways? Can you elaborate a bit on why it's okay to catch and do nothing here?

Reply:
Uhm, okay I try.
When you call the "binary" with `--container`, some validation logic inside the `ArgvInput` of symfony throws that exception as the `--container` option is configured to require a value. 
This is, because we are actually accessing it in `ContainerResolver::resolve`.

As I dont want to duplicate that code which renders that exception to the console, I simply suppress the exception as it will be properly shown as well when executing `$app->run`.

So there are 3 options to me

1. catch exception, render the error as it would be done by the application by duplicating that code
2. catch exception, suppress it and let `$app->run` do its job
3. do not catch exception and have that default PHP "uncaught exception" error in CLI


Due to my laziness, I prefer the current solution (no. 2)

Reply:
That's fine - just document what exactly you're suppressing here, so that future maintainers aren't left wondering why the code is the way it is. :)